toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -253,6 +262,44 @@ func (b *LiveOSIsoBuilder) stageIsoMakerInitrdArtifacts(writeableRootfsDir, isoM
 	return nil
 }
 
+// getGrubBinary
+//
+// returns the name of the grub binary for the current architecture.
+func getGrubBinary() (string, error) {
+
+	archBootBinaries := map[string]string{
```

Comment:
Move this mapping to the top of the file. (Make it a global.)

Similar comments below.

Reply:
It might be beneficial to use a settings struct.

e.g.

```golang
type BootFilesArchConfig struct {
    grubBinary string,
    grubNoPrefixBinary string,
    bootBinary string,
    …
}

var (
    bootloaderFilesConfig = map[string]BootFilesArchConfig{ … }
)